<?xml version="1.0" encoding="utf-8" standalone="no"?>
<test-run id="2" testcasecount="2" result="Failed" total="2" passed="0" failed="2" inconclusive="0" skipped="0" asserts="0" engine-version="3.8.0.0" clr-version="4.0.30319.42000" start-time="2019-01-03 01:36:42Z" end-time="2019-01-03 01:36:44Z" duration="2.152091">
  <command-line><![CDATA["D:\BuildTools\NUnit.org\nunit-console\nunit3-console.exe" .\bin\Debug\Hivemind.Api.Tests.dll --where "class == Hivemind.Api.Tests.Features.UsersFeature"]]></command-line>
  <filter>
    <class>Hivemind.Api.Tests.Features.UsersFeature</class>
  </filter>
  <test-suite type="Assembly" id="0-1003" name="Hivemind.Api.Tests.dll" fullname="D:\Source\Hivemind\Hivemind\Hivemind.Api.Tests\bin\Debug\Hivemind.Api.Tests.dll" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2019-01-03 01:36:43Z" end-time="2019-01-03 01:36:44Z" duration="1.132282" total="2" passed="0" failed="2" warnings="0" inconclusive="0" skipped="0" asserts="0">
    <environment framework-version="3.10.1.0" clr-version="4.0.30319.42000" os-version="Microsoft Windows NT 10.0.17763.0" platform="Win32NT" cwd="D:\Source\Hivemind\Hivemind\Hivemind.Api.Tests" machine-name="TOMBSTONE" user="Greg" user-domain="TOMBSTONE" culture="en-CA" uiculture="en-US" os-architecture="x64" />
    <settings>
      <setting name="DisposeRunners" value="True" />
      <setting name="WorkDirectory" value="D:\Source\Hivemind\Hivemind\Hivemind.Api.Tests" />
      <setting name="ImageRuntimeVersion" value="4.0.30319" />
      <setting name="ImageTargetFrameworkName" value=".NETFramework,Version=v4.6.2" />
      <setting name="ImageRequiresX86" value="False" />
      <setting name="ImageRequiresDefaultAppDomainAssemblyResolver" value="False" />
      <setting name="NumberOfTestWorkers" value="4" />
    </settings>
    <properties>
      <property name="_PID" value="12732" />
      <property name="_APPDOMAIN" value="domain-" />
    </properties>
    <failure>
      <message><![CDATA[One or more child tests had errors]]></message>
    </failure>
    <test-suite type="TestSuite" id="0-1004" name="Hivemind" fullname="Hivemind" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2019-01-03 01:36:43Z" end-time="2019-01-03 01:36:44Z" duration="1.097298" total="2" passed="0" failed="2" warnings="0" inconclusive="0" skipped="0" asserts="0">
      <failure>
        <message><![CDATA[One or more child tests had errors]]></message>
      </failure>
      <test-suite type="TestSuite" id="0-1005" name="Api" fullname="Hivemind.Api" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2019-01-03 01:36:43Z" end-time="2019-01-03 01:36:44Z" duration="1.096526" total="2" passed="0" failed="2" warnings="0" inconclusive="0" skipped="0" asserts="0">
        <failure>
          <message><![CDATA[One or more child tests had errors]]></message>
        </failure>
        <test-suite type="TestSuite" id="0-1006" name="Tests" fullname="Hivemind.Api.Tests" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2019-01-03 01:36:43Z" end-time="2019-01-03 01:36:44Z" duration="1.096499" total="2" passed="0" failed="2" warnings="0" inconclusive="0" skipped="0" asserts="0">
          <failure>
            <message><![CDATA[One or more child tests had errors]]></message>
          </failure>
          <test-suite type="TestSuite" id="0-1007" name="Features" fullname="Hivemind.Api.Tests.Features" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2019-01-03 01:36:43Z" end-time="2019-01-03 01:36:44Z" duration="1.096469" total="2" passed="0" failed="2" warnings="0" inconclusive="0" skipped="0" asserts="0">
            <failure>
              <message><![CDATA[One or more child tests had errors]]></message>
            </failure>
            <test-suite type="TestFixture" id="0-1000" name="UsersFeature" fullname="Hivemind.Api.Tests.Features.UsersFeature" classname="Hivemind.Api.Tests.Features.UsersFeature" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2019-01-03 01:36:43Z" end-time="2019-01-03 01:36:44Z" duration="1.089463" total="2" passed="0" failed="2" warnings="0" inconclusive="0" skipped="0" asserts="0">
              <properties>
                <property name="Description" value="Users" />
              </properties>
              <failure>
                <message><![CDATA[One or more child tests had errors]]></message>
              </failure>
              <output><![CDATA[-> Using app.config
]]></output>
              <test-case id="0-1001" name="AddNewUser" fullname="Hivemind.Api.Tests.Features.UsersFeature.AddNewUser" methodname="AddNewUser" classname="Hivemind.Api.Tests.Features.UsersFeature" runstate="Runnable" seed="461001166" result="Failed" label="Error" start-time="2019-01-03 01:36:44Z" end-time="2019-01-03 01:36:44Z" duration="0.490031" asserts="0">
                <properties>
                  <property name="Description" value="Add new user" />
                </properties>
                <failure>
                  <message><![CDATA[System.UriFormatException : Invalid URI: Invalid port specified.]]></message>
                  <stack-trace><![CDATA[   at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   at Hivemind.Api.Tests.Context.Post[TRequest,TResponse,TError](String path, TRequest request) in D:\Source\Hivemind\Hivemind\Hivemind.Api.Tests\Context.cs:line 51
   at Hivemind.Api.Tests.Steps.UserSteps.WhenIAddAUserAsFollows(Table table) in D:\Source\Hivemind\Hivemind\Hivemind.Api.Tests\Steps\UserSteps.cs:line 38
   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)
   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.ExecuteStepMatch(BindingMatch match, Object[] arguments)
   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.ExecuteStep(IContextManager contextManager, StepInstance stepInstance)
   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.OnAfterLastStep()
   at Hivemind.Api.Tests.Features.UsersFeature.ScenarioCleanup()
   at Hivemind.Api.Tests.Features.UsersFeature.AddNewUser() in D:\Source\Hivemind\Hivemind\Hivemind.Api.Tests\Features\Users.feature:line 10]]></stack-trace>
                </failure>
                <output><![CDATA[When I add a user as follows:
  --- table step argument ---
  | Email               | Password |
  | testuser3@email.com | hunter2  |
-> error: Invalid URI: Invalid port specified.
When I retrieve token with using:
  --- table step argument ---
  | Email               | Password |
  | testuser3@email.com | hunter2  |
-> skipped because of previous errors
Then I should receive a token
-> skipped because of previous errors
]]></output>
              </test-case>
              <test-case id="0-1002" name="TryToGetUserInformationWithoutLoggingIn" fullname="Hivemind.Api.Tests.Features.UsersFeature.TryToGetUserInformationWithoutLoggingIn" methodname="TryToGetUserInformationWithoutLoggingIn" classname="Hivemind.Api.Tests.Features.UsersFeature" runstate="Runnable" seed="1305296131" result="Failed" label="Error" start-time="2019-01-03 01:36:44Z" end-time="2019-01-03 01:36:44Z" duration="0.006639" asserts="0">
                <properties>
                  <property name="Description" value="Try to get user information without logging in" />
                </properties>
                <failure>
                  <message><![CDATA[System.UriFormatException : Invalid URI: Invalid port specified.]]></message>
                  <stack-trace><![CDATA[   at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   at Hivemind.Api.Tests.Context.Get[TResponse](String path) in D:\Source\Hivemind\Hivemind\Hivemind.Api.Tests\Context.cs:line 34
   at Hivemind.Api.Tests.Steps.UserSteps.WhenIGetUserInformation() in D:\Source\Hivemind\Hivemind\Hivemind.Api.Tests\Steps\UserSteps.cs:line 45
   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)
   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.ExecuteStepMatch(BindingMatch match, Object[] arguments)
   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.ExecuteStep(IContextManager contextManager, StepInstance stepInstance)
   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.OnAfterLastStep()
   at Hivemind.Api.Tests.Features.UsersFeature.ScenarioCleanup()
   at Hivemind.Api.Tests.Features.UsersFeature.TryToGetUserInformationWithoutLoggingIn() in D:\Source\Hivemind\Hivemind\Hivemind.Api.Tests\Features\Users.feature:line 14]]></stack-trace>
                </failure>
                <output><![CDATA[When I get user information
-> error: Invalid URI: Invalid port specified.
Then I should receive an error as follow:
  --- table step argument ---
  | ErrorMessage |
  | Dang!        |
-> skipped because of previous errors
]]></output>
              </test-case>
            </test-suite>
          </test-suite>
        </test-suite>
      </test-suite>
    </test-suite>
  </test-suite>
</test-run>